cluster/gce/gci/configure.sh

```diff
@@ -158,6 +158,9 @@ function install-kube-binary-config {
   fi
 
   cp "${KUBE_HOME}/kubernetes/LICENSES" "${KUBE_HOME}"
+  cp "${KUBE_HOME}/kubernetes/kubernetes-src.tar.gz" "${KUBE_HOME}"
```

Comment:
Where does the script download `kubernetes-src.tar.gz`? This is not included in `kubernetes-release/release/v1.3.0-beta.2/kubernetes-server-linux-amd64.tar.gz `. My cluster upgrade failed because of this: `./cluster/gce/upgrade.sh -M v1.3.0-beta.2`